hazelcast/src/main/java/com/hazelcast/config/AtomicLongConfig.java

```diff
@@ -37,7 +42,9 @@ public AtomicLongConfig(AtomicLongConfig config) {
     @Override
     public String toString() {
         return "AtomicLongConfig{"
-                + fieldsToString()
+                + "name='" + name + '\''
```

Comment:
It is safer to handle fields in the unified way in the class where they are declared. Duplication is evil.

Reply:
yeah, I know what you mean abuot duplication, but there's some cut-off point. This method was just too much for me, in terms of value-added vs. complexity to change sth later on.

Reply:
You could just put the implementation into the parent:
```java
    public String toString() {
        return getClass().getSimpleName() + "{"
                + "name='" + name + '\''
...
```

Reply:
What if I add a new field in the child's class. Will you then create some abstract methods to add to the initial toString that the child can override? It's basic stuff, string concatenation, each class can handle it by itself. I'm fine with removing duplication where the complexity is there, but there's no complexity in here. Just some over-complexification of simple things.

Reply:
Once the fields differs, I would use the original approach with `fieldsToString`. Nevermind, this is just nitpicking. I can live with the suggested change.

Reply:
@kwart ok, let me add the toString as you want it.

Reply:
Fixed ;)